src/main/java/org/openrewrite/java/migrate/apache/commons/lang/ApacheCommonsStringUtils.java

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.apache.commons.lang;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+
+public class ApacheCommonsStringUtils {
+    public static class Chop {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.chop(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return s.substring(0, s.length() - 1);
+        }
+    }
+
+    public static class DefaultString {
+        @BeforeTemplate
+        String before(String s) {
+            return StringUtils.defaultString(s);
+        }
+
+        @AfterTemplate
+        String after(String s) {
+            return Objects.toString(s);
+        }
+    }
+
+    public static class Equals {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.equals(s, "string");
+        }
+
+        @AfterTemplate
+        boolean after(String s) {
+            return s.equals("string");
+        }
+    }
+
+    public static class IsEmpty {
+        @BeforeTemplate
+        boolean before(String s) {
+            return StringUtils.isEmpty(s);
+        }
+
+        @AfterTemplate
+        boolean after(String s) {
+            return s.isEmpty();
```

Comment:
I think this requires a null check too
```suggestion
            return s ==null || s.isEmpty();
```